docs/src/main/sphinx/release/release-467.md

```diff
@@ -0,0 +1,105 @@
+# Release 467 (4 Dec 2024)
+
+## General
+
+* Change default protocol for internal communication to HTTP/1.1. ({issue}`24281`)
```

Comment:
Do we need to add more here @wendigo .. or say something like "Change default back.. and if you want HTTP/2 .."

Reply:
It's broken in some cases so until Jetty is updated there is no point in enabling it back